scripts/stubtest/allowlist_todo_django60.txt

```diff
@@ -202,15 +202,12 @@ django.forms.ClearableFileInput.use_fieldset
 django.forms.models.BaseModelForm.validate_constraints
 django.forms.renderers.Jinja2DivFormRenderer
 django.forms.widgets.ClearableFileInput.use_fieldset
-django.tasks
-django.tasks.backends
-django.tasks.backends.base
-django.tasks.backends.dummy
-django.tasks.backends.immediate
-django.tasks.base
 django.tasks.checks
-django.tasks.exceptions
-django.tasks.signals
+django.tasks.default_task_backend
```

Comment:
Question: why are these items in `todo.txt`?
- If these are expected errors, move them to regular `allowlist.txt` (without `todo`) and comment why they are here
- If they are just not fixed yet, what is blocking us from fixing them?

Reply:
I moved the task decorator to `allowlist.txt` with proper reasoning.

I left the `default_task_backend` in todo mirroring the other ConnectionProxy instances. I also left enqueue method as todo for now.